.goreleaser.yml

```
@@ -9,19 +9,20 @@ builds:
     goarch:
     - amd64
     - 386
+    - arm64
     main: .
     env:
       - CGO_ENABLED=0
     binary: event-generator
 
 dockers:
-  -
+  - use: buildx
     goos: linux
     goarch: amd64
     dockerfile: Dockerfile
     image_templates:
-      - "falcosecurity/event-generator:latest"
-      - "falcosecurity/event-generator:{{ .Version }}"
+      - "falcosecurity/event-generator:latest-amd64"
+      - "falcosecurity/event-generator:{{ .Version }}-amd64"
```

```suggestion
      - "falcosecurity/event-generator:latest"
      - "falcosecurity/event-generator:{{ .Version }}"
```

Could we avoid using multiple tags? AFAIK, we should be able to push multiple architectures on the same tag (ie. `latest`). Perhaps, we should also use an option like this :point_down: to inform goreleaser: 

```
  # Extra flags to be passed down to the manifest push command.
  # Defaults to empty.
  push_flags:
  - --platform amd64
```
(I haven't checked in detail, just guessing).